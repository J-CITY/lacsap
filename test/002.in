31231.315533
