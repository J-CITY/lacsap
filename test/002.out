ERROR: -1002 6 14
